libs/agno/agno/os/app.py

```diff
@@ -176,6 +179,7 @@ def __init__(
 
         self.telemetry = telemetry
         self.tracing = tracing
+        self.tracing_db = tracing_db
```

Comment:
if tracing_db is set, automatically set `tracing=True`. That also means make `tracing: Optional[bool] = None` so that you know whether it was set or not.